ted_sws/notice_metadata_processor/adapters/notice_metadata_extractor.py

```diff
@@ -320,3 +415,43 @@ def extract_code_and_value_from_element(element: ET.Element) -> EncodedValue:
     if element is not None:
         return EncodedValue(code=extract_attribute_from_element(element=element, attrib_key="CODE"),
                             value=extract_text_from_element(element=element))
+
+def extract_code_from_element(element: ET.Element) -> EncodedValue:
+    """
+    Extract code from text value from an element in the XML structure
+    :param element:
+    :return:
+    """
+    if element is not None:
+        return EncodedValue(code=extract_text_from_element(element=element),
+                            value=extract_text_from_element(element=element))
+
+def parse_xml_manifestation(xml_manifestation: XMLManifestation) -> ET.Element:
+    """
+    Parsing XML manifestation and getting the root
+    :return:
+    """
+    xml_manifestation_content = xml_manifestation.object_data
+    return ET.fromstring(xml_manifestation_content)
+
+
+def normalised_namespaces_from_xml_manifestation(xml_manifestation: XMLManifestation) -> Dict:
+    """
+    Get normalised namespaces from XML manifestation
+    :return:
+    """
+    namespaces = dict([node for _, node in ET.iterparse(source=StringIO(xml_manifestation.object_data),
+                                                        events=['start-ns'])])
+
+    namespaces["manifestation_ns"] = namespaces.pop("") if "" in namespaces.keys() else ""
```

Comment:
define and use constants instead of string literals

Reply:
I did this